src/main/java/org/apache/sling/dynamicinclude/generator/types/JsiGenerator.java

```diff
@@ -39,7 +39,7 @@
  */
 @Component
 public class JsiGenerator implements IncludeGenerator {
-    private static final String TEMPLATE_FILENAME = "generators/jquery.html";
+    private static final String TEMPLATE_FILENAME = "generators/javascript.html";
```

Comment:
@SBPrakash - Looks like you changed the template, but didn't delete the jquery one, could you delete that as well?

Reply:
It's removed now.